### PR TITLE
Add Maven AspectJ plugin build cache configuration

### PR DESCRIPTION
**Pull request by guylabs**

### CODE REVIEW

File: .github/workflows/maven-build-caching-samples-verification.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
             -c "$GITHUB_SHA" \
             -p "build-caching-maven-samples" \
             -g "verify" \
-            -a "-Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug -B" \
+            -a "-Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug -Dorg.slf4j.simpleLogger.log.gradle.goal.fingerprint=trace -B" \
```
Comment (runningcode) [inline .github/workflows/maven-build-caching-samples-verification.yml:34]: oh, what does this flag do and how does it help here?

**Review by runningcode — APPROVED** [review]
In general looks good, just a question about the extra logging.

File: build-caching-maven-samples/pom.xml [hunk 2]
```diff
@@ -8,20 +8,21 @@
   <name>gradle-enterprise-maven-caching-sample</name>
 
   <modules>
```
Comment (erichaagdev) [inline build-caching-maven-samples/pom.xml:10]: Love the alphabetization. 😄 👍

Comment (guylabs) [reply build-caching-maven-samples/pom.xml:10]: Learned from the best ;)

File: .github/workflows/maven-build-caching-samples-verification.yml (hunk 1 above)
Comment (guylabs) [reply .github/workflows/maven-build-caching-samples-verification.yml:34]: This is just for now to find the issue with the action that fails and the validation scripts. Will revert it later once all is fixed.

Comment (guylabs) [reply .github/workflows/maven-build-caching-samples-verification.yml]: Reverted in https://github.com/gradle/gradle-enterprise-build-config-samples/pull/546/commits/64b57eb538c932ee0b3c9810cb58abd7ef00b343

**Pull request merged**